gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -127,6 +127,10 @@ spec:
    enabled: true
    port: 3000
 ----
++
+Applying the YAML configuration snippet enables the readiness API to respond to `GET` requests:
++
+`\http://<flex-gateway-url>:<port-configured>/probes/readiness?allowEnvoyErrors=<true|false>&allowAPIErrors=<true|false>&allowPolicyErrors=<true|false>`
```

Comment:
@glenn-rodgers-sf The problem this introduces is those query params aren’t documented in that Docker tab; they’re documented in the k8s tab. So someone who selects the Docker tab and reads your new content won’t know what all those params are. The customer will have to click the other tab to find out, which isn’t very usable.